jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ArtifactHandler.kt

```diff
@@ -110,7 +110,7 @@ class ArtifactHandler(private val project: Project, private val clientAdaptor: G
                 codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
                 codeTransformRunTimeLatency = calculateTotalLatency(apiStartTime, apiEndTime),
                 codeTransformJobId = job.id,
```

Comment:
Updated name to be more consistent across our events. This change is in vscode and codeTransformTotalByteSize is a "nice to have" not a business requirement.